server/main.go

```diff
@@ -12,14 +15,13 @@ var upgrader = websocket.Upgrader{
 }
 
 func main() {
-	// TODO
-	// open a goroutine for each user
-	// need a mutex for the board?
 	// matchmaking and new game making
 	playerId := 0
 	board := NewBoard(19)
```

Comment:
this `19` is a magic number and should be replaced

eventually we could even have a massive board with loads of players hehe